src/components/Layouts/index.js

```diff
@@ -45,6 +47,28 @@ class Layout extends React.Component {
     });
   }
 
+  handleSearchClick = (isSearchOpen) => {
```

Comment:
We should call this on keypress as well

Reply:
When I tab to the magnifying glass it open on enter. Do you mean that it should close on a keypress ?